ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/ExplicitDisclosureIT.scala

```diff
@@ -570,7 +571,15 @@ final class ExplicitDisclosureIT extends LedgerTestSuite {
             testContext.disclosedContract,
             testContext.disclosedContract.update(
               // Set distinct contract id
-              _.modify(_.withContractId("00" * 32 + "ff"))
```

Comment:
Fixed: the old one would fail in Canton with another error due to invalid contract id.